src/Backend/Core/Installer/Data/install.sql

```diff
@@ -86,25 +86,25 @@ CREATE  TABLE IF NOT EXISTS `backend_navigation` (
   `selected_for` TEXT NULL ,
   `sequence` INT(11) NOT NULL ,
   PRIMARY KEY (`id`) )
- ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=1 ;
+ ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci AUTO_INCREMENT=1 ;
 
 
 CREATE TABLE IF NOT EXISTS `hooks_queue` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `module` varchar(255) NOT NULL,
+  `module` varchar(255) CHARACTER SET utf8 NOT NULL,
   `callback` text NOT NULL,
   `data` text ,
   `status` enum('busy','error','queued') NOT NULL DEFAULT 'queued',
   `created_on` datetime NOT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=MyISAM  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=1 ;
+) ENGINE=MyISAM  DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci AUTO_INCREMENT=1 ;
 
 
 CREATE TABLE IF NOT EXISTS `hooks_subscriptions` (
-  `event_module` varchar(255) NOT NULL,
-  `event_name` varchar(255) NOT NULL,
-  `module` varchar(255) NOT NULL,
+  `event_module` varchar(255) CHARACTER SET utf8 NOT NULL,
+  `event_name` varchar(255) CHARACTER SET utf8 NOT NULL,
+  `module` varchar(255) CHARACTER SET utf8 NOT NULL,
```

Comment:
Also because the unique key is larger than the max. allowed 1000 bytes here... :( So if I'd use utf8mb4 here, the query would throw an error.
utf8 takes 3 bytes and utf8mb4 takes 4 bytes
Unique key size with utf8mb4 used: 100 * 4 + 100 * 4 + 100 * 4 = 1200 > 1000

So I changed the 3 fields used in the unique key to utf8, and now it would only take 3 bytes: 100 * 3 + 100 * 3 + 100 * 3 = 900 < 1000
(I only need to change two of them to utf8 actually but I chose to do all 3).